spec/e2e/accessibility.test.js

```diff
@@ -1,18 +1,20 @@
 import { describe, before, after, test, it } from 'node:test';
 import assert from 'node:assert';
-import { relative, dirname } from 'node:path';
-import glob from 'fast-glob';
+import { readFileSync } from 'node:fs';
 import puppeteer from 'puppeteer';
 import { AxePuppeteer } from '@axe-core/puppeteer';
 import { createServer } from 'http';
 import handler from 'serve-handler';
 import getPort from 'get-port';
 
-const paths = glob
-  .sync('_site/**/index.html')
-  .map((path) => dirname(relative('_site', path)))
-  .filter((path) => path !== '.')
-  .map((path) => `/${path}/`);
+// only test canonical paths
+const paths = (() => {
+  const file = readFileSync('./_site/sitemap.xml', { encoding: 'utf8' });
+  const pathMatch = file.matchAll(/[.gov|:4000](\/[A-Za-z-]+\/?[A-Za-z-]*\/)<\/loc/g);
+  const foundPaths = Array.from(pathMatch).map((a) => a[1]);
+  foundPaths.unshift('/');
+  return foundPaths;
```

Comment:
Rather than hardcoding the port here (which can change with local developer configuration or CLI flags), maybe we parse it as a URL and use the pathname? Also avoids having to manually add in `/`.

```suggestion
  const pathMatch = file.matchAll(/<loc>(.+?)<\/loc>/g);
  return Array.from(pathMatch).map(([, url]) => new URL(url).pathname));
```

Reply:
hm, this catches `/risk.json`, which is not an HTML file and causes the tests to fail. I like this direction, I'll dig into it more.